### PR TITLE
fix(config-ui): some bugs

### PR DESCRIPTION
**Pull request by mintsweet**
### Summary
1. bp not properly filterd.
2. rate limit lost default value.
3. add default value for transformation.
4. adjust the rules for project name.
5. not reset value after project create.
6. adjust the list page parameter pageSize.
7. endpoint subLabel error for gitlab.
8. add plugin customize.
9. add default value for use-operator's formatReason.
10. optimize the creation of webhook in the project.

### CODE REVIEW

File: config-ui/src/pages/project/detail/use-project.ts [hunk 1]
```diff
@@ -55,7 +55,7 @@ export const useProject = (name: string) => {
   }, []);
 
   const handleUpdate = async (newName: string, enableDora: boolean) => {
-    if (!/^\w+$/.test(newName)) {
+    if (!/^\w|-\/+$/.test(newName)) {
```
Comment (likyh) [inline config-ui/src/pages/project/detail/use-project.ts:58]: It not works, I think it means /(^\w)|(-(\/+))$/.

File: config-ui/src/pages/project/home/use-project.ts [hunk 2]
```diff
@@ -53,7 +53,7 @@ export const useProject = <T>({ name, enableDora, onHideDialog }: Props) => {
   }, []);
 
   const handleSave = async () => {
-    if (!/^\w+$/.test(name)) {
+    if (!/^\w|-\/+$/.test(name)) {
```
Comment (likyh) [inline config-ui/src/pages/project/home/use-project.ts:56]: Same above

Comment (mintsweet) [reply config-ui/src/pages/project/home/use-project.ts]: thanks.

File: config-ui/src/pages/project/detail/use-project.ts (hunk 1 above)
Comment (mintsweet) [reply config-ui/src/pages/project/detail/use-project.ts]: thank.

**Review by likyh — APPROVED** [review]

**Pull request merged**